unit_tests/Makefile

```diff
@@ -297,7 +293,7 @@ ULIBDIR =
 
 # List all user libraries here
 ULIBS = -lm
-ULIBS += -lgcov --coverage
```

Comment:
is this going to not break code coverage?

Reply:
running the `ci_gcov.sh` script on my mac generated an identical html page to what's currently here: https://rusefi.com/docs/unit_tests_coverage/index.html